packages/lesswrong/lib/collections/comments/schema.ts

```diff
@@ -875,7 +875,58 @@ const schema: SchemaType<"Comments"> = {
     canRead: ['guests'],
     canUpdate: ['sunshineRegiment', 'admins'],
     canCreate: ['members', 'sunshineRegiment', 'admins'],
-  } 
+  },
+
+  doppelComments: {
+    // Resolved by doppelComments resolver
+    type: Array,
+    optional: true,
+    canRead: ['guests'],
+    canCreate: ['admins'],
+    canUpdate: ['admins'],
+    hidden: true,
+  },
+
+  'doppelComments.$': {
+    type: Object,
+  },
+
+  ownDoppelCommentVote: resolverOnlyField({
+    type: Object,
+    graphQLtype: 'DoppelCommentVote',
+    canRead: ['guests'],
+    resolver: async (comment: DbComment, args: void, context: ResolverContext) => {
+      const { DoppelCommentVotes, currentUser } = context;
+      if (!currentUser) return null;
+      return DoppelCommentVotes.findOne({commentId: comment._id, userId: currentUser._id}) ?? null;
+    },
+    sqlResolver: ({field, currentUserField}) => `(
+      SELECT ROW_TO_JSON(dc.*)
+      FROM "DoppelCommentVotes" dc
+      WHERE dc."commentId" = ${field("_id")} AND dc."userId" = ${currentUserField("_id")}
+    )`
+  }),
+  doppelCommentVoteChoices: resolverOnlyField({
+    type: Array,
+    graphQLtype: '[String]',
+    canRead: ['guests'],
+    resolver: async (comment: DbComment, args: void, context: ResolverContext) => {
+      const { DoppelCommentVotes } = context;
+      const doppelCommentVotes = await DoppelCommentVotes.find({commentId: comment._id, type: 'vote'}).fetch();
```

Comment:
hold on, doesn't this need to also filter for the current user's userId?  (And then we should still do an `accessFilterMultiple`, even if we have a `checkAccess` on `DoppelCommentVotes`.

Reply:
Or if I'm misunderstanding and this is intended to be public info, then the `checkAccess` is probably wrong.. idk, need to talk about it.